### PR TITLE
Docs: Twig allowed attributes

### PR DESCRIPTION
**Pull request by adamkudrna**
- Document all allowed HTML attributes in all components
- Unify the syntax of boolean props

Partially relates to #957.

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Text/README.md [hunk 1]
```diff
@@ -29,11 +29,12 @@ Without lexer:
...
-| `emphasis`    | `regular`, `bold`, `italic`                 | `regular` | no       | Emphasis of the text |
-| `size`        | [Size Extended dictionary][dictionary-size] | `medium`  | no       | Size of the text     |
+| Prop name     | Type                                        | Default   | Required | Description                                                    |
+| ------------- | ------------------------------------------- | --------- | -------- | -------------------------------------------------------------- |
+| `elementType` | `string`                                    | `p`       | no       | HTML tag to render                                             |
+| `emphasis`    | `regular`, `bold`, `italic`                 | `regular` | no       | Emphasis of the text                                           |
+| `size`        | [Size Extended dictionary][dictionary-size] | `medium`  | no       | Size of the text                                               |
+| `translate`   | [`yes`, `no`]                               | `null`    | no       | Set to `no` to disable machine translation of the text content |
```
[the remark this answers is not in the document]
Comment (dlouhak) [reply packages/web-twig/src/Resources/components/Text/README.md:37]: `[]` will increase readability, but I wouldn't want to be the one who will edit the documentation. 🙊

[the remark this answers is not in the document]
Comment (dlouhak) [reply packages/web-twig/src/Resources/components/Text/README.md:37]: Btw, it looks like [empty string is also allowed](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/translate#:~:text=the%20following%20values%3A-,empty%20string%20or%20yes,-%2C%20which%20indicates%20that).

```suggestion
| `translate`   | [`yes`, `no`, ``]                               | `null`    | no       | Set to `no` to disable machine translation of the text content |
```

[the remark this answers is not in the document]
Comment (literat) [reply packages/web-twig/src/Resources/components/Text/README.md:37]: If I may... I would prefer unix/linux-like argument documenting style. E.g.: `man [-adho] [-t | -w] [-M manpath] [-P pager] [-S mansect]`, so for this ```[`yes` | `no` | ``]```. But as far as I am re-reading it after myself I see there can be a problem with MD table formatting. Never mind. Rubber 🦆 programming.  @dlouhak has a point for readability 👍

**Review by literat — APPROVED** [review]
Cool 👍

File: packages/web-twig/src/Resources/components/Text/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Text/README.md:37]: If you would like to have my opinion, then know, that I don't care 😂 both (comma separated and with or without brackets) look good to me.

**Review by pavelklibani — APPROVED** [review]

File: packages/web-twig/src/Resources/components/Text/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/Text/README.md:37]: > `[]` will increase readability

Alright then 👍🏻.

> there can be a problem with MD table formatting

The pipe needs escaping:

```md
[`yes` \| `no` \| `''`]
```

Result:

image

I dislike the escaping in the raw MD a little but, but let's try it, it's only in a few places anyway.

[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/Text/README.md:37]: Done in a separate PR: #1001

**Pull request merged**